src/library.js

```diff
@@ -1449,13 +1498,17 @@ const createPilingJs = (rootElement, initOptions = {}) => {
           if (pile.items.length !== state.piles[id].items.length) {
             updatePileItems(pile, id);
             updatePileStyle(pile, id);
+            changedPiles.push(id);
+            stateUpdates.add('arrange');
```

Comment:
I don't think we actually need this as `updatedPiles.length > 0` will be the same as `stateUpdates .has('arrange')`